### PR TITLE
FALCON-1982 : Configuration steps for HBase standalone mode

### PR DESCRIPTION
**Pull request by vramachan**
Step to run HBase in standalone mode to support titan backend.

### CODE REVIEW

File: docs/src/site/twiki/Configuration.twiki [hunk 1]
```diff
@@ -362,7 +363,45 @@ We recommend that in the startup config the tablename for titan storage be named
 *.falcon.graph.storage.hbase.table=falcon_titan
 </verbatim>
 
-Permissions
+---++++Starting standalone HBase for testing
+
+HBase can be started in stand alone mode for testing as a backend for Titan. The following steps outline the config changes required:
+<verbatim>
+1. Build Falcon as below to package hbase binaries
+   $ export MAVEN_OPTS="-Xmx1024m -XX:MaxPermSize=256m" && mvn clean assembly:assembly -Phadoop-2,package-standalone-hbase
```
Comment (vrangan) [inline docs/src/site/twiki/Configuration.twiki]: hadoop2 no longer exists

**Comment by vrangan** [comment]
Fix and then ship

File: docs/src/site/twiki/Configuration.twiki [hunk 2]
```diff
@@ -368,7 +368,7 @@ We recommend that in the startup config the tablename for titan storage be named
 HBase can be started in stand alone mode for testing as a backend for Titan. The following steps outline the config changes required:
 <verbatim>
 1. Build Falcon as below to package hbase binaries
-   $ export MAVEN_OPTS="-Xmx1024m -XX:MaxPermSize=256m" && mvn clean assembly:assembly -Phadoop-2,package-standalone-hbase
+   $ export MAVEN_OPTS="-Xmx1024m -XX:MaxPermSize=256m" && mvn clean assembly:assembly package-standalone-hbase
```
Comment (bvellanki) [inline docs/src/site/twiki/Configuration.twiki]: It should be 

"$ export MAVEN_OPTS="-Xmx1024m -XX:MaxPermSize=256m" && mvn clean assembly:assembly -Ppackage-standalone-hbase"

**Comment by bvellanki** [comment]
+1 will commit shortly

**Pull request closed without merging**